plugin_runner/sandbox.py

```diff
@@ -1,12 +1,18 @@
+from __future__ import annotations
+
 import ast
```

Comment:
@aduane @djantzen please give this one file an additional review; it now does what it should